### ISSUE TITLE
Unable to deploy updated website on my sub domains

### ISSUE THREAD

**Issue by coreybruce** [BitBalloon/bitballoon-js#2]
Hi I am having issues updating my webpage on BitBaloon, I drag in the folder containing the index.html and other files like usual but it gets stuck on processing. I checked the network activity and I can see its going through checking a bunch of files but never completes like it use to, PLEASE FIX!

**Comment by Kaisaurus** [comment]
I had the same problem so uploaded files to git and used their newer service [netlify](netlify.com). Work much better!

**Comment by coreybruce** [comment]
Yeah problem was solved by reuploading also I already was using there netifly website.
